resources/splunk_app.rb

```diff
@@ -162,30 +163,40 @@ class PackagedApp < SplunkApp
 
   action :install do
     return unless !version || changed?(:version)
+    @converge_actions = nil # Clear resource updates by property change
```

Comment:
Can you provide a bit more context on what this change is for? The existing comment doesn't really help me much.